### ISSUE TITLE
How Do I Get This Working On Devuan (Debian 10, etc)?

### ISSUE THREAD

**Issue by RennisDitchie** [B00merang-Project/System-4#1]
Hey there, 
I read through the README, and understand that that the dependencies include:
GTK+ 3.20 (or higher)
Murrine and Pixmap theme engines

However, I'm really not sure of how to get this working on a Debian based distro aka Devuan.

I believe I would have to utilize a 'chimaera' or a testing package to get this to work, but I have no clue on how to start.

I don't want to utilize GNOME, so I'm going with the 'Mate' Desktop Environment.

I believe the "Murrine" and "Pixmap" dependencies are "themes" but I don't see them in the package managers for Devuan, even on the testing branches.

Can you please link to either of these three dependencies to get me in the right direction.

I'm just a bit lost on how to actually get the dependencies (and don't mind if I have to build them from source or something).

I am the type of person that has used "AwesomeWM" and even Emacs Window Manager, but can't seem to figure out how to actually get this working. I ask because I just want this working on my "JustWorks" desktop, and thought this would be really cool.

Thanks.

**Comment by dominichayesferen** [comment]
Murrine and Pixmap are only for GTK2, and given that GTK2 is end of life, you might not need to worry about that pretty soon.

**Comment by Elbullazul** [comment]
@dominichayesferen some apps still use GTK2 (older releases of the GIMP, for example).

@RennisDitchie I believe the package name for Debian is gtk2-engines or something like that.

**Comment by RennisDitchie** [comment]
Hey there,
Thanks for letting me know that these dependencies are for GTK2, which is end of life.

Understood.

However, I did a git clone for the repo, and placed its contents into ~/.themes/System-4

The folder itself contains "gtk-2.0", "gtk-3.0", "index.theme", and the "LICENSE" and "README.md" files.

The thing is, I'm using MATE desktop on Devuan Beowulf.

When I go to System > Preferences > Look And Feel > Appearance > I then try to select the "System 4" theme, I'm getting a "Exclamation Point: This theme will not look as intended because the required GTK+ theme 'System 4' is not installed", related screenshot:
https://ibb.co/CtZ3Hgy

Yet, when I click "Install" and navigate to the "~/.themes/System-4" folder, I can't select the related "index.theme" file as intended, screenshot:
https://ibb.co/SPhD8PG

Thanks for the help so far.

**Comment by Elbullazul** [comment]
you have a dash in the path, but in the index it's a space. If you rename the 'System-4' directory to 'System 4', it should be ok

**Comment by RennisDitchie** [comment]
Ah, this might be something you guys want to rename on the Git repo in that case cause I pulled it from your git repo (though I understand because probably whitespace characters for the folder itself would end up weird on the final release)

Anyway, I renamed '~/.themes/System-4' to:
'~/.themes/System 4'

But, I get a SLIGHTLY different issue when re-attempting to add it via System > Preferences > Look And Feel > Appearance.

I'm getting a message "This theme will not look as intended because the required window manager theme 'System 4' is not installed.

Related screenshot:
https://ibb.co/wW9wq1H

When I attempt to "Install" it, same thing, I go to '~/.themes/System 4' but MATE doesn't see anything to "install" despite the "index.theme" file being present.

**Comment by Elbullazul** [comment]
yes, this theme doesn't have a window manager theme yet. But it's relatively easy to make one. I'll try to make something this week, I'll keep you updated

**Comment by RennisDitchie** [comment]
Ok cool, thanks, I'll bookmark this project's GitHub page, sweet.

By the way, I'm using MATE Desktop Environment 1.20.4 if it helps with Devuan Beowulf (Release 3).

Thanks again, looking forward to it.

Can't say I didn't learn anything from this because I'm looking forward to getting this on my system as I miss Macintosh a TON.

By the way, I am going to also put a "Feature request to do this same "Window Manager Theme" action for the "Mac OS 9" theme as well:
https://github.com/B00merang-Project/Mac-OS-9-Classic

Random question while I wait, can I request themes as well?

I would LOVE to have an Amiga style theme, that would be sick.

Anyway, will put in related feature requests on respective pages, thanks again. Would be interested in how this works to possibly contribute as this kind of thing is pretty sweet, never saw a project like this, so let me know how that works.

Related Feature Requests That I Created As  A Result:
https://github.com/B00merang-Project/Mac-OS-9-Classic/issues/1

EDIT: Didn't know how to create a Feature Request for an "Amiga" theme, but I guess this would be a cool revision to look out for.

**Comment by Elbullazul** [comment]
you can open a theme suggestion issue here: https://github.com/B00merang-Project/Azurra_framework.

Amiga was always something that I wanted to try but never gave it time, as I felt it was a bit too 'niche' this project. Also, please include the amigaOS version you want, as there are a couple of different styles I believe.

**Comment by RennisDitchie** [comment]
Hey, any update on this?

Just checking in.

**Comment by Elbullazul** [comment]
Hello,

I haven't really thought about it, but it is still something I want to work on.

I usually do the dev work on a VM but I have problems with it currently and haven't taken time to fix them. Hope to get it working again soon
